core/src/test/java/io/parsingdata/metal/data/callback/CallbackTest.java

```diff
@@ -66,20 +67,20 @@ public void testHandleCallback() throws IOException {
 
         final Token cho = cho("cho", ONE, TWO);
         final Token sequence = seq("seq", cho, ONE);
-        final TokenCallbackList callbacks = TokenCallbackList
-            .create(new TokenCallback(ONE, countingCallback))
-            .add(new TokenCallback(TWO, countingCallback))
-            .add(new TokenCallback(cho, countingCallback))
-            .add(new TokenCallback(sequence, countingCallback));
-        final Environment environment = new Environment(new InMemoryByteStream(new byte[] { 2, 1 }), new Callbacks(callbacks));
+        final Callbacks callbacks =
+            NONE.add(ONE, countingCallback)
```

Comment:
I've added a factory method `create()` to `Callbacks` that returns `NONE`. I've kept `NONE` public, because in cases where you want to state that you want no callbacks, passing `NONE` actually looks better than passing `Callbacks.create()` IMHO.